docs/book/orders/checkout.rst

```diff
@@ -72,7 +72,7 @@ How to perform the Addressing Step programmatically?
 
 Firstly if the **Customer** is not yet set on the Order it will be assigned depending on the case:
 
-* An already logged in **User** - the Customer is set for the Order using the `CartBlamerListener <https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/CoreBundle/EventListener/CartBlamerListener.php>`_, that determines the user basing on the event.
+* An already logged in **User** - the Customer is set for the Order using the `ShopCartBlamerListener <https://github.com/Sylius/Sylius/blob/1.13/src/Sylius/Bundle/ShopBundle/EventListener/ShopCartBlamerListener.php>`_, that determines the user basing on the event.
```

Comment:
```suggestion
* An already logged in **User** - the Customer is set for the Order using the `ShopCartBlamerListener <https://github.com/Sylius/Sylius/blob/1.12/src/Sylius/Bundle/ShopBundle/EventListener/ShopCartBlamerListener.php>`_, that determines the user basing on the event.
```

As we target this PR to `1.12` let's use the same version when linking to the file. We'll change the version to `1.13` while upmerging (at least we'll try to remember about it).

Reply:
yes you are right, i've just pushed the suggestion